### ISSUE TITLE
missing directories in .gitignore

### ISSUE THREAD

**Issue by burgerrg** [newrelic/c-sdk#11]
v1.1.0 on Ubuntu 18.04.3 after a clean checkout and make:
```
$ git status
On branch master
Your branch is up to date with 'origin/master'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	vendor/newrelic/bin/
	vendor/newrelic/pkg/
```

Please add these directories to `.gitignore`. Thanks!

**Comment by Fahmy-Mohammed** [comment]
Hi Bob! Thanks for finding this issue. We will add it to the `.gitignore` and it will be included the next C SDK release.

**Comment by Fahmy-Mohammed** [comment]
Implemented in [4f9d766a7](https://github.com/newrelic/c-sdk/commit/4f9d766a7a02dac7fd395858076662c5c05f0069) and released in C-SDK version 1.2.0. Thank you for your feedback!
